### ISSUE TITLE
Notice: VoiceOver must enter a group to access the link, why?

### ISSUE THREAD

**Issue by FelFly** [Esri/calcite-components#646]
Sorry for all the accessibility stuff all at once, I had to review a bunch of the components we were including in our app. 

I noticed that when navigating through a Notice with VoiceOver you do not hear the link read out as a link item, you hear the group and group title rather than any indication that there is a link. You also have to then navigate into the group before you can interact with the link or hear the link text. What is the reason for this? Why is this in a group? It seems like this would be better suited as a link without the group wrapper so that the VoiceOver user doesn't have to enter into a static group in order to interact with an element.

![image](https://user-images.githubusercontent.com/7389593/83780722-87df1400-a65b-11ea-8e2b-3fc3246854f9.png)

https://esri.github.io/calcite-components/?path=/story/notice--simple

**Comment by macandcheese** [comment]
No problem - thank you for opening them as we want to get them fixed. 

The rendered element ends up being an anchor link - I wonder if it has to do with the link being slotted content - we don't intentionally group it otherwise. I assume this might be the reason for some of the other issues as well. Will investigate!

**Comment by jcfranco** [comment]
Installed. 

@FelFly Could you help verify this one w/ `@esri/calcite-components@next`?

**Comment by FelFly** [comment]
Verified - works as I'd expect.

**Comment by jcfranco** [comment]
Thanks for verifying, @FelFly! 🙌
